types/lhr-lite.d.ts

```diff
@@ -70,49 +70,10 @@ declare global {
         /** An explanation of audit-related issues encountered on the test page. */
         explanation?: string;
         /** Extra information provided by some types of audits. */
-        details?: Audit.MetricDetails | Audit.OpportunityDetails;
+        details?: never;
```

Comment:
I'm going to start deleting `lhr-lite-d.ts` since its use didn't pan out, so starting by snipping off everything starting here